src/components/Country.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import './styles/Country.css';
+
+function Country({ country }) {
+  const { name, flag } = country;
+  return (
+    <>
+      <div className="card">
+        <NavLink
+          to={`/details/${name}`}
+          className="details-link"
+          data-testid="details-link"
+        >
+          <div className="image">
+            <img src={flag} alt="country" />
+          </div>
+          <h1 className="country-name">{name}</h1>
+        </NavLink>
+      </div>
+    </>
+  );
+}
```

Comment:
Hello, can you follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-\(Free-PSDs\)) including:

* Typography: font face, size, and weight
* Layout: composition and space between elements.

NB: You need to have two cards in the horizontal line. Also, provide a small arrow on each card, and reduce the font sizes of each name of the country. Follow the original design

- Your App: 
![Capture](https://github.com/siddghosh108/React-Country-Capstone/assets/63972670/9d9e6855-09ca-4eaf-90ff-a9a4a559b6b6)
- Original design
![Capture1](https://github.com/siddghosh108/React-Country-Capstone/assets/63972670/f6bf77ed-e759-4636-a431-6b2b92230af4)

Reply:
I have followed the design of the following reference:
https://explore-the-countries.onrender.com/

If you open the link you will find that my design is exactly like the one that i have followed, so i will request you to allow me to follow this design which i have even mentioned in my README.md file

Reply:
- [ ] As requested by the previous reviewer in this [comment](https://github.com/siddghosh108/React-Country-Capstone/pull/1#discussion_r1355132099) kindly follow the given design guidelines

- Typography: font face, size, and weight
- Layout: composition and space between elements.
NB: You need to have two cards in the horizontal line(Mobile size). Also, provide a small arrow on each card, and reduce the font sizes of each name of the country. Follow the original design


- [ ] According to the previous reviewer, your font face has to be changed, which I agree with. the given font is Lato.
![image](https://github.com/siddghosh108/React-Country-Capstone/assets/81932478/5d39e286-d7f4-48f5-a9ab-cb44c6db610e)

- [ ] The layout is also something you should consider fixing. This is a mobile app and therefore in mobile view as low as 375px, you should have two cards not three as is in your design

Your design
![image](https://github.com/siddghosh108/React-Country-Capstone/assets/81932478/29474611-4df4-469f-8c45-85e92f6513b5)

Given design
![image](https://github.com/siddghosh108/React-Country-Capstone/assets/81932478/dbdedc0e-c5a3-476d-9034-454183ad66ac)

Reply:
Changed the Homepage,css file inside ./pages/styles to get the following UI for the mobile version

![image](https://github.com/siddghosh108/React-Country-Capstone/assets/131663359/cb7516e8-b310-4f8c-84a8-8751206d683e)

Also changed the countries-container class to include the Lato font
.countries-container {
  display: grid;
  grid-template-columns: repeat(3, minmax(0, 1fr));
  justify-content: center;
  width: 100%;
  font-family: 'Lato', sans-serif; /* Change the font to Lato */
}

Reply:
You did well implementing some changes suggested by the previous reviewer, but you still  have some more to go that was mentioned but wasn't properly implemented 👍 : 

- I'd suggest you ensure a consistency in your design, as suggested by the previous reviewer about using `Lato` font, while you implemented it for the cards and the countries, you used a different font for the heading which is `Let's explore countries`, please kindly implement the right font for the heading
- As suggested  by the previous reviewer and also visible in the design, kindly include the arrows shown in the design given 😄 
- Lastly, I'd suggest you reduce the font size for the countries in the card , something around `17px` should be fine so as to ensure right font size suggested by the previous reviewer 😄 

Your design :
![image](https://github.com/siddghosh108/React-Country-Capstone/assets/20537712/d3be5e34-34ed-4b0b-a6bb-fe7833c76d19)

Design given :

![image](https://github.com/siddghosh108/React-Country-Capstone/assets/20537712/2fb50a01-8e01-4643-b58b-4c025e73ca18)